### PR TITLE
28 torrent debug screen

### PR DESCRIPTION
**Pull request by ChelseaGuan**
closes issue #28

### CODE REVIEW

File: detoks/src/main/java/nl/tudelft/trustchain/detoks/TorrentFragment.kt [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        val hopCountTV = view.findViewById<TextView>(R.id.hopCountTextView)
+        val watchedTV = view.findViewById<TextView>(R.id.watchedTextView)
+        val durationTV = view.findViewById<TextView>(R.id.durationTextView)
+        val likesTV = view.findViewById<TextView>(R.id.likesTextView)
+        val timesSeenTV = view.findViewById<TextView>(R.id.timesSeenTextView)
+        val uploadDateTV = view.findViewById<TextView>(R.id.uploadDateTextView)
+
+        val errorTV = view.findViewById<TextView>(R.id.errorTextView)
+
+        val passedName = "electricsheep-flock-248-7500-9"
```
Comment (Floor97) [inline detoks/src/main/java/nl/tudelft/trustchain/detoks/TorrentFragment.kt:38]: Does it still use only this hardcoded value or dynamically get it from the view?

File: detoks/src/main/java/nl/tudelft/trustchain/detoks/DeToksFragment.kt [hunk 2]
```diff
@@ -56,7 +56,7 @@ class DeToksFragment : BaseFragment(R.layout.fragment_detoks) {
         super.onViewCreated(view, savedInstanceState)
 
         viewPagerVideos.adapter = VideosAdapter(torrentManager)
-        viewPagerVideos.currentItem = 0
+            viewPagerVideos.currentItem = 0
```
Comment (Floor97) [inline detoks/src/main/java/nl/tudelft/trustchain/detoks/DeToksFragment.kt:59]: Why the indent?

**Review by Floor97 — CHANGES_REQUESTED** [review]
Looks good! However, instead of the hardcoded strings in the code in `TorrentFragment`, I would just add a separate TextView in the `fragment_torrent.xml` with a reference to a string in the `strings.xml`, and put the string in `strings.xml`. For example "Hop count: " doesn't have to be in the code in `TorrentFragment`

File: detoks/src/main/java/nl/tudelft/trustchain/detoks/TorrentFragment.kt (hunk 1 above)
Comment (ChelseaGuan) [reply detoks/src/main/java/nl/tudelft/trustchain/detoks/TorrentFragment.kt:38]: Oups, good catch! Should be dynamic now

**Pull request merged**